main.css

```diff
@@ -320,6 +353,33 @@ ul {
   padding: 0;
 }
 
+/* This class were added here because Linters give me the next error
+1000:1 Expected selector ".popup-section ul li" to come before selector ".grid-section .secondary-card no-descending-specificity
+li"
+ 1050:1 Expected selector ".text-container ul li" to come before selector ".grid-section .secondary-card no-descending-specificity
+li" */
+
+.popup-section ul li {
```

Comment:
- [ ] You may consider removing the space between the ```See Live``` and ```See source``` buttons so your design is as close as possible to the Figma design

![Screenshot_20220512_210131](https://user-images.githubusercontent.com/25227022/168158875-2fed82cc-0f33-40f5-8ff2-3d09c63a4ccd.png)